File: anika.dovob/src/main/java/com/example/anika/homework/chapter8/Team.java
package com.example.anika.homework.chapter8;

import jakarta.persistence.*;

@Entity(name = "Ch8Team")
@Table(name = "team_ch8")
public class Team {

    @Id
    @Column(name = "team_id")
    private Long id;

    @Column(nullable = false)
    private String name;

    protected Team() {
    }

    public Team(Long id, String name) {
        this.id = id;
        this.name = name;
    }

    public Long getId() {
        return id;
    }

    public String getName() {
        return name;
    }
}

File: gracie.lee/src/main/kotlin/homework/chapter_8/Team.kt
package homework.chapter_8

import jakarta.persistence.*

// 1) 엔티티 스켈레톤 - Team: id(PK), name(not null)
@Entity(name = "Ch8TeamKt")
@Table(name = "team_ch8_kt")
class Team(
    @Id
    @Column(name = "team_id")
    val id: Long? = null,  // @GeneratedValue 없음 → 테스트에서 ID 직접 지정 (2번 요구사항)

    @Column(nullable = false)
    var name: String = ""
)
